lib/setup_computer.rb

```diff
@@ -0,0 +1,76 @@
+class SetupComputer
+  attr_reader :board, :ships
+
+  def initialize(board, ships)
+    @board = board
+    @ships = ships
+  end
+
+  def max_index
+    @board.cells.size - 1
+  end
+
+  def random_index
+    (0..max_index).to_a.sample
+  end
+
+  def row_arr
+    @board.cells.keys
+  end
+
+  def col_arr
+    ["A1", "B1", "C1", "D1",
+    "A2", "B2", "C2", "D2",
+    "A3", "B3", "C3", "D3",
+    "A4", "B4", "C4", "D4"]
+  end
+
```

Comment:
Refactor later if we want to scale the board size.

Reply:
Good ID of possible future refactor